playground/nextjs/src/pages/render-notification/index.tsx

```diff
@@ -10,11 +10,26 @@ export default function Home() {
         {...novuConfig}
         renderNotification={(notification) => {
           return (
-            <div className="flex gap-2 flex-nowrap items-start self-stretch my-1 p-2 hover:bg-slate-200">
-              <div className="rounded-full w-8 h-8 overflow-hidden border border-cyan-200">Avatar</div>
+            <div
```

Comment:
added this example to be sure that the `renderNotification` callback is called with the new value